src/coreComponents/mesh/generators/InternalWellGenerator.cpp

```diff
@@ -246,8 +255,8 @@ void InternalWellGenerator::findPolylineHeadNodeIndex()
     real64 const currentZcoord = m_polyNodeCoords[inode][2];
 
     GEOS_THROW_IF( !(currentZcoord < headZcoord),
-                   "Error in the topology of well '" << getName()
-                                                     << "' since we found a well node that is above the head node",
+                   getDataContext() << ": Error in the topology since we found a well node that" <<
+                   " is above the head node",
```

Comment:
What is the number of characters you allow per line?
In `geos` it's 200.

Reply:
It is not what I read in [Code Style](https://geosx-geosx.readthedocs-hosted.com/en/latest/docs/sphinx/developerGuide/Contributing/CodeStyle.html) docs.